### DISCUSSION TITLE
RfE: German Localization

### DISCUSSION THREAD

**Discussion by RainerBielefeld** [aljawaid/TagManager#6] [General]
Currently there is no German TUI Translation.
I can do until weekend.

**Comment by aljawaid** [comment]
hello, there is no need to create an issue for it... just create a discussion if you like and then submit the changes.  I will convert this issue into a discussion.  Translations are never issues unless the strings or functions are incorrect.

**Reply by RainerBielefeld** [reply]
### OT: 
> Translations are never issues  

Well, that's the philosophy here. Most Software projects to which I have contributed, and also dictionaries, have a different PoV ...

Bu of course, I also can do such announcements as Discussion in future

**Reply by aljawaid** [reply]
hehe I understand what you mean.... but I think you need to be mindful that not every language can be translated/serviced.

If Kanboard had public data to show which languages were the most popular based on installations etc and which plugins were the most popular then I would say developers should translate for them languages.

In the open source world, its all about keeping a balance.  I translate all my plugins for English UK only because some American English words are quite different and it is good practise for me as a learner to be able to code like that.  I can only translate other languages by using Google etc and that is time consuming and its an assumption (without asking a local native) whether the translation is truly correct.

I do understand your point of view though. But like @creecros once said, translations should be up to the users.  I agree, as that way, we have an idea of which languages are being used by the plugins.  There is simply no other data for us.

**Comment by aljawaid** [comment]
@RainerBielefeld this template contains all the plugin strings: 
https://github.com/aljawaid/TagManager/tree/master/Locale
